src/exchange_adapters/base.ts

```diff
@@ -214,103 +187,6 @@ export abstract class BaseExchangeAdapter {
    */
   abstract fetchTrades(): Promise<Trade[]>
```

Comment:
As we discussed, I'm convinced that it's worth keeping this around since we could use it again in the future. Could you add a comment explaining that it's not currently in use?

Reply:
Done :)